networks/movement/movement-full-node/src/node/tasks/execute_settle.rs

```diff
@@ -152,10 +152,17 @@ where
 			&& da_height % self.settlement_config.settle.settlement_super_block_size == 0
 		{
 			info!("Posting block commitment via settlement manager");
-			match self.settlement_manager.post_block_commitment(commitment).await {
-				Ok(_) => {}
-				Err(e) => {
-					error!("Failed to post block commitment: {:?}", e);
+			match &self.settlement_manager {
+				Some(settlement_manager) => {
+					match settlement_manager.post_block_commitment(commitment).await {
+						Ok(_) => {}
+						Err(e) => {
+							error!("Failed to post block commitment: {:?}", e);
+						}
+					}
+				}
+				None => {
+					error!("Settlement manager not initialized");
```

Comment:
If it was disabled in the config, it should not be an error?

Reply:
Yes, I would prefer to do this. But, because of how this was written, there isn't a super clean way to do this. Would take a little longer than I had.

Reply:
My concern is that we'll get log spam on every block, but the node will not otherwise fail. But since this would clearly be a misconfiguration, the exact failure mode does not matter much.